lang/cs/Org.Apache.REEF.Wake/Remote/Impl/StreamingRemoteManager.cs

```diff
@@ -77,15 +76,15 @@ public WritableRemoteManager(IPAddress localAddress, int port, ITcpPortProvider
         /// <summary>
         /// Constructs a DefaultRemoteManager. Does not listen for incoming messages.
         /// </summary>
-        /// <param name="injector">The injector to pass arguments to incoming messages</param>
+        /// <param name="streamingCodec">Streaming codec</param>
         [Obsolete("Use IRemoteManagerFactory.GetInstance() instead.", false)]
-        public WritableRemoteManager(IInjector injector)
+        public StreamingRemoteManager(IStreamingCodec<T> streamingCodec)
```

Comment:
yupp.. i m gonna do that .

On Sun, Jul 5, 2015 at 11:33 AM, Markus Weimer <notifications@github.com>
wrote:

> In lang/cs/Org.Apache.REEF.Wake/Remote/Impl/StreamingRemoteManager.cs
> <https://github.com/apache/incubator-reef/pull/261#discussion_r33897408>:
>
> >          [Obsolete("Use IRemoteManagerFactory.GetInstance() instead.", false)]
> > -        public WritableRemoteManager(IInjector injector)
> > +        public StreamingRemoteManager(IStreamingCodec<T> streamingCodec)
>
> Yes, but the other constructor takes a ITcpPortProvider *and* a port. We
> need one constructor that's clean.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-reef/pull/261/files#r33897408>.
>